src/components/Nav/index.js

```diff
@@ -16,10 +18,11 @@ NavItem.propTypes = {
 }
 
 export const Nav = ({ ariaLabel, children, className}) => (
-  <nav className={`nav ${className && className}`} aria-label={ariaLabel}>
-    <ul className="nav__list">
+  <nav className={`nav ${className && className} : ""`} aria-label={ariaLabel}>
```

Comment:
@helrond can you take a look at this again? I'm not sure I pulled this together correctly when fixing the merge conflicts with base with the switch from class to functional component. I'm also seeing that the My List link comes and goes at different screen sizes before the Dropdown kicks in.

Reply:
Yeah that should be 
```
{`nav ${className ? className : ""}`}
```

In terms of the nav, I think there may be an issue with the design where the lower end of the break points isn't accounted for. I may have to tweak padding/margins or something.